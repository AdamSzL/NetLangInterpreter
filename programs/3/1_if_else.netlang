if true {
    print "if worked"
}
else {
    print "else worked"
}


if false {
    print "if worked"
}
else if true {
    print "else if worked"
}
else {
    print "else worked"
}


if false {
    print "if worked"
}
else if false {
    print "else if worked"
}
else {
    print "else worked"
}


if false {
    print "if worked"
}
else if false {
    print "first else if"
}
else if true {
    print "second else if"
}
else {
    print "else worked"
}