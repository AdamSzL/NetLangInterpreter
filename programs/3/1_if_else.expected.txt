if worked
else if worked
else worked
second else if